frontend/src/Movie/Details/MovieDetailsLinks.js

```diff
@@ -23,7 +24,7 @@ function MovieDetailsLinks(props) {
           kind={kinds.INFO}
           size={sizes.LARGE}
         >
-          TMDb
+          {translate('TMDb')}
```

Comment:
doesn't _need_ translated

Reply:
_shouldn't_ _need_ to be

seems future proof?